### PR TITLE
Feature/message viewer

### PR DESCRIPTION
**Pull request by orndorffgrant**
# what does this do
oh man so much - sorry this is way too big for a normal PR
- lists files
- upload files
- display messages
- fetch messages
- highlight content of messages

# how to test
make sure you have the getMessage endpoint merged for the backend first
go through entire user flow
- signup
- login
- upload file
- upload anotha one
- click "explore" next to a file
- see first message in file
- hover over things
- click on things in list and in raw, make sure correct things highlight
- click the up arrow next to message num at top right, make sure the next message is fetched and displayed (untested as of Friday 4pm)

### CODE REVIEW

File: src/views/MessagePage.tsx [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+import { inject, observer } from 'mobx-react';
+import { FILE_STORE, IFileStore } from '../stores/fileStore';
+
+const useStyles = makeStyles(theme => ({
+  container: {
+    display: 'grid',
+    height: '100vh',
+    gridTemplateColumns: '1fr 2fr 1fr 1fr',
+    gridTemplateRows: 'auto 3fr 4fr',
+    // gridTemplateAreas: `
```
Comment (Perry5) [inline src/views/MessagePage.tsx]: Maybe we can delete commented out code if we don't need it?

File: src/views/MessagePage.tsx [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+    gridArea: 'raw',
+    fontFamily: 'monospace',
+    overflow: 'scroll',
+    boxShadow: 'inset 0 10px 10px -10px #b0b0b0',
+    marginTop: theme.spacing(1),
+    padding: theme.spacing(2, 1),
+    fontSize: '1.2em',
+    lineHeight: '1.3em',
+    whiteSpace: 'nowrap',
+    // resize: 'vertical',
```
Comment (Perry5) [inline src/views/MessagePage.tsx]: same here

File: src/views/MessagePage.tsx (hunk 1 above)
Comment (orndorffgrant) [reply src/views/MessagePage.tsx]: yeah I was playing around with different layouts and allowing resizing, but this is easy to recreate if we want to adjust layout in the future, I'll delete it.

File: src/views/MessagePage.tsx (hunk 2 above)
Comment (orndorffgrant) [reply src/views/MessagePage.tsx]: same as above

**Pull request merged**